Sources/StreamChatSwiftUI/ChatChannel/MessageList/ImageAttachmentView.swift

```diff
@@ -93,21 +93,23 @@ public struct ImageAttachmentContainer<Factory: ViewFactory>: View {
     }
 }
 
-public struct AttachmentTextView: View {
+public struct AttachmentTextView<Factory: ViewFactory>: View {
     @Injected(\.colors) private var colors
     @Injected(\.fonts) private var fonts
 
+    var factory: Factory
     var message: ChatMessage
     let injectedBackgroundColor: UIColor?
 
-    public init(message: ChatMessage, injectedBackgroundColor: UIColor? = nil) {
+    public init(factory: Factory, message: ChatMessage, injectedBackgroundColor: UIColor? = nil) {
```

Comment:
We should put a default parameter `DefaultViewFactory.shared` to avoid breaking changes here.

Reply:
we should put the default param here